variables.tf

```diff
@@ -0,0 +1,374 @@
+  description = "Redis snapshot window"
+  default     = "mon:10:00-mon:12:00"
+}
+
+variable "redis_auto_minor_version_upgrade" {
+  description = "Redis allow auto minor version upgrade"
+  default     = "true"
+}
+
+variable "redis_at_rest_encryption_enabled" {
+  description = "Redis encrypt storage"
+  default     = "false"
+}
+
+variable "redis_transit_encryption_enabled" {
+  description = "Redis encrypt transit TLS"
+  default     = "false"
+}
+
+variable "redis_replicas_count" {
+  description = "Number of replica nodes in each node group"
+  default     = 1
+}
+
+variable "redis_allowed_subnet_cidrs" {
+  description = "List of CIDRs/subnets which should be able to connect to the Redis cluster"
+  type        = "list"
+  default     = ["127.0.0.1/32"]
+}
+
+variable "redis_subnet_tag_filter" {
+  description = "The Map to filter the subnets of the VPC where the Redis component of the Microservice resides"
+  default     = {}
+}
+
+# -------------------------------------------------------------------------------------------------
+# RDS
+# -------------------------------------------------------------------------------------------------
+variable "rds_enabled" {
+  description = "Set to false to prevent the module from creating any rds resources"
+  default     = "false"
+}
+
+variable "rds_identifier_override" {
+  description = "Redis cluster ID. Use only lowercase, numbers and -, _., only use when it needs to be different from var.name"
+  default     = ""
+}
+
+variable "rds_allowed_subnet_cidrs" {
+  description = "List of CIDRs/subnets which should be able to connect to the RDS instance"
+  type        = "list"
+  default     = ["127.0.0.1/32"]
+}
+
+# -------------------------------------------------------------------------------------------------
+# RDS instance engine
+# -------------------------------------------------------------------------------------------------
+variable "rds_engine" {
+  description = "RDS instance engine"
+  type        = "string"
+  default     = "mysql"
+}
+
+variable "rds_major_engine_version" {
+  description = "RDS instance major engine version"
+  type        = "string"
+  default     = "5.7"
+}
+
+variable "rds_engine_version" {
+  description = "RDS instance engine version"
+  type        = "string"
+  default     = "5.7.19"
+}
+
+variable "rds_family" {
+  description = "Parameter Group"
+  type        = "string"
+  default     = "mysql5.7"
+}
+
+# -------------------------------------------------------------------------------------------------
+# RDS instance node type and replication
+# -------------------------------------------------------------------------------------------------
+variable "rds_node_type" {
+  description = "VM type which should be taken for nodes in the RDS instance"
+  type        = "string"
+  default     = "db.t3.micro"
+}
+
+variable "rds_multi_az" {
+  description = "Replication settings"
+  type        = "string"
+  default     = true
```

Comment:
There is mixed usage of `true` and `"true"`. Let's please consolidate to use one or the other

Reply:
@cytopia I believe that in some of these cases (if there's an urgent need for something to be delivered), we can make these changes ourselves given that we're on common ground. WDYT?

Reply:
Valid point. Difference between Anton's modules and Cloudposse is that Cloudposse at the moment is trying to have fixed typing to be ahead with the 0.12 migration game.